Sources/FaunaDB/Value.swift

```diff
@@ -300,6 +300,37 @@ fileprivate func escapeObject(with: String, object: [String: Value]) -> JsonType
     ])
 }
 
+/// Represents a binary blob returned by the server.
+/// [Reference](https://fauna.com/documentation/queries#values-special_types).
+public struct BytesV: ScalarValue, AsJson {
+
+    public var value: Data
```

Comment:
Initially I implemented using array of bytes, however it was causing some issues when I was trying to extract values or assign to a variable of `[UInt8]`. It end up using the array version of get (`public func get<T>(asArrayOf field: Field<T>) throws -> [T] `) so it was treating the `BytesV` as a collection.